virttest/utils_test/libvirt.py

```diff
@@ -3577,3 +3578,40 @@ def get_disk_alias(vm, source_file=None):
                 logging.info("Ignore error of source attr getting for file: %s" % e)
                 pass
     return None
+
+
+def check_cmd_expected(check_cmd, matched, is_expected=True):
```

Comment:
we can reuse the existing API,
https://github.com/avocado-framework/avocado-vt/blob/e07894bf447cf83ff34eeffecc68ebb99a5d7892/virttest/utils_test/libvirt.py#L1514-L1532

Reply:
yes，i think so.

Reply:
I will close this pr